api/grpc_proto/trace-sampling-manager-service/trace-sampling-manager.proto

```diff
@@ -7,6 +7,8 @@ option go_package = "grpc_proto/trace_sampling_manager";
 // The Trace Sampling Manager service definition.
 service TraceSamplingManager {
     rpc SetHostsToTrace (HostsToTraceRequest) returns (Empty){}
+    rpc SetHostsToRemove (HostsToRemoveRequest) returns (Empty){}
```

Comment:
this shouldn't be called SetHostsToRemove, be cause we don't set it, we remove it from the set list.
Better to call it RemoveFromHostsToTrace or UntraceHosts